src/components/NestedAccordionMenu/MenuItem.js

```diff
@@ -0,0 +1,102 @@
+import React, { Fragment, useState } from "react";
+import List from "@material-ui/core/List";
+import ListItem from "@material-ui/core/ListItem";
+import ExpandMoreIcon from "@material-ui/icons/ExpandMore";
+import ExpandLessIcon from "@material-ui/icons/ExpandLess";
+import Collapse from "@material-ui/core/Collapse";
+import ListItemIcon from "@material-ui/core/ListItemIcon";
+import ListItemText from "@material-ui/core/ListItemText";
+import { makeStyles } from "@material-ui/core/styles";
+const useStyles = makeStyles(theme => ({
+  sidebarItem: {
+    display: "flex",
+    justifyContent: "space-between",
+    alignItems: "center",
+  },
+  sidebarItemContent: {
+    whiteSpace: "nowrap",
+    textOverflow: "ellipsis",
+    overflow: "hidden",
+    display: "flex",
+    alignItems: "center",
+    width: "100%",
+  },
+  collapse: {
+    background: "#f4f1f1",
+  },
+}));
+function MenuItem({ depthStep = 10, depth = 0, expanded, item, ...rest }) {
```

Comment:
I usually prefer to use default props instead

```
MenuItem.defaultProps = {
    depth:  0,
    depthStep: 10
}
```

Though I also find that dephStep should be a constant and not changeable, so I’d remove it from the props.